src/test/ui/macros/issue-54441.stderr

```diff
@@ -1,8 +1,8 @@
-error: expected one of `crate`, `fn`, `pub`, `static`, or `type`, found `let`
+error: expected one of `crate`, `fn`, `pub`, `static`, or `type`, found keyword `let`
   --> $DIR/issue-54441.rs:3:9
    |
 LL |         let
-   |         ^^^ unexpected token
+   |         ^^^ expected one of `crate`, `fn`, `pub`, `static`, or `type` here
```

Comment:
In the change above this one the word `here` is removed in this PR (“ ^^ expected `fn` here”), while it is still present in this change.

Is that as intended?

Reply:
Updated these as well.